packages/core/content-releases/admin/src/components/ReleaseActionMenu.tsx

```diff
@@ -179,6 +180,35 @@ const ReleaseActionEntryLinkItem = ({
   );
 };
 
+/* -------------------------------------------------------------------------------------------------
+ * EditReleaseItem
+ * -----------------------------------------------------------------------------------------------*/
+interface EditReleaseItemProps {
+  releaseId: Release['id'];
+}
+
+const EditReleaseItem = ({ releaseId }: EditReleaseItemProps) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <StyledMenuItem>
+      <Link
+        as={NavLink}
+        // @ts-expect-error TODO: This component from DS is not using types from NavLink
+        to={`/plugins/content-releases/${releaseId}`}
+        startIcon={<Icon as={Pencil} width={3} height={3} />}
+      >
+        <Typography variant="omega">
+          {formatMessage({
+            id: 'content-releases.content-manager-edit-view.edit-release',
+            defaultMessage: 'Edit release',
```

Comment:
you're gonna get a million of these labels, is it worth doing `Edit {release.name} release`?

Reply:
Not sure If I understand, the link looks like this on the Edit View for each release attached so I think there is no space to add the release name. Maybe you want to add the name in a aria-label for better accessibility?

<img width="333" alt="Screenshot 2024-02-23 at 14 53 41" src="https://github.com/strapi/strapi/assets/23050694/c7021f8d-6ea2-4f94-8e0f-4e4f683c181a">

Reply:
ah sorry, i thought this was an icon button in a table because i saw the `releases.map` my bad, no you're right this is probably fine 😄